app/components/home-page.js

```diff
@@ -24,6 +31,17 @@ export default Ember.Component.extend({
     });
   },
 
+  isEmptyOrganization: (function() {
+    const organization = this.get("organization");
+    const isOwner = this.get("me.org.is_owner")
+    if(isOwner) {
+      const orgName = organization.selected.get("name");
+      if(!orgName) {
+        return true;
+      }
+    }
+  }).property("me.org.is_owner"),
```

Comment:
Can we move this to use the recommended syntax? Using Ember.computed instead of .property. This method is discouraged and might be removed in future. https://guides.emberjs.com/release/configuring-ember/disabling-prototype-extensions/